developers/weaviate/model-providers/friendliai/generative.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Generative AI
+sidebar_position: 51
+image: og/docs/integrations/provider_integrations_friendliai.jpg
+# tags: ['model providers', 'friendliai', 'generative', 'rag']
+---
+
+# FriendliAI Generative AI with Weaviate
+
+import BetaPageNote from '../_includes/beta_pages.md';
+
+<BetaPageNote />
+
+:::info Added in `v1.27.0`
+:::
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyConnect from '!!raw-loader!../_includes/provider.connect.py';
+import TSConnect from '!!raw-loader!../_includes/provider.connect.ts';
+import PyCode from '!!raw-loader!../_includes/provider.generative.py';
+import TSCode from '!!raw-loader!../_includes/provider.generative.ts';
+
+Weaviate's integration with FriendliAI's APIs allows you to access open source and their models' capabilities directly from Weaviate.
```

Comment:
I don't think this is a correct sentence (access open source models?).

Suggestion:
Weaviate's integrations with FriendliAI's APIs allow you to access open-source models on Friendli and utilize models' capabilities directly from Weaviate.